src/test/components/__snapshots__/TooltipMarker.test.js.snap

```diff
@@ -3282,91 +3282,6 @@ exports[`TooltipMarker renders tooltips for various markers: IPCOut-120 1`] = `
 </div>
 `;
 
-exports[`TooltipMarker renders tooltips for various markers: IPCOut-121 1`] = `
```

Comment:
These tests were having a wrong assumption that there could be multiple markers of the same IPC message in a thread. Now they are removed and fixed!